pkg/abstractions/image/build.go

```diff
@@ -299,7 +299,7 @@ func (b *Builder) Build(ctx context.Context, opts *BuildOpts, outputChan chan co
 				errMsg = err.Error() + "\n"
 			}
 
-			time.Sleep(1 * time.Second) // Wait for logs to be passed through
```

Comment:
This is not long enough if the user's connection is slow due to internet and/or distance from our datacenter